R/sgdnet.R

```diff
@@ -0,0 +1,103 @@
+#' Fit a Generalized Linear Model with Elasticnet Regularization
+#'
+#' @param x input matrix
+#' @param y response variable
+#' @param family reponse type
+#' @param alpha elasticnet mixing parameter
+#' @param lambda regularization strength
+#' @param intercept whether to fit an intercept or not
+#' @param maxit maximum number of effective passes (epochs)
+#' @param standardize whether to standardize `x` or not -- ignored when
+#'   `intercept == TRUE`.
```

Comment:
Why is this ignored if we have an intercept? We might still want to standardize `X` so that the columns have the same norm (this is often done to put the variables "on the same footing").

Reply:
Sorry, that's a mistake in the documentation. It is the other way around. No standardization is done when there is no intercept. This is how they do it in scikit-learn's Ridge. I don't know why though, but I just translated it over for now.